newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
```

Comment:
I think this file should be in gitignore, it has license_key

Reply:
adding to git ignore! @rfagg805